src/components/GraduatesSummer2019/index.js

```diff
@@ -0,0 +1,199 @@
+import React from 'react'
+import MVIMG_20190720_190912 from '../../img/MVIMG_20190720_190912.jpg'
```

Comment:
Interesting! You'll have to help me understand what is going on here.

Reply:
ok. I will show. it works because of the package called 'responsive-loader'. it returns object with src, and srcset. and I could not make it work on static pages. that is why I copy pasted the img foler to src folder. it only works in react.